config/locales/views/wise_items/en.yml

```
@@ -0,0 +1,142 @@
+---
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)